### ISSUE TITLE
Fix the shebangs

### ISSUE THREAD

**Issue by danascape** [sahil-sagwekar2652/GitHub-Automation-scripts#20]
* is the character sequence "#!" at the beginning of a script file in Unix-like operating systems, including Linux. It is followed by the path to the interpreter or command that should be used to execute the script.
* It's purpose is to specify the interpreter to execute the file

* The file supports python3+ and not older versions like python2, while some systems which have python2 as default, the script will not execute on these systems.

**Comment by Akshayp02** [comment]
hello sir I'm GSSOC23 contributor can you assign me this issue i would like to explore and resolve it.

**Comment by bshantherishenoy** [comment]
hello sir I'm GSSOC23 contributor can you assign me this issue i would like to explore and resolve it.

**Comment by adityavpawar** [comment]
Please assign me this issue so that I can resolve it under GSSOC-23

**Comment by sahil-sagwekar2652** [comment]
Hey everyone, the issues are being assigned on a first-come-first-serve basis. In the meanwhile, check out the other issues, or feel free to create your own. Also, don't make a PR without raising an issue and getting it assigned to you. If the original assigned isn't active or not working on the issue, then I'll assign it to the next contributor.

One contributor will be assigned to one issue at a time, so please take up one issue at a time, don't comment on every issue.

**Comment by sahil-sagwekar2652** [comment]
@Akshayp02 any updates? Let me know or I'll have to unassign you.

**Comment by Akshayp02** [comment]
> @Akshayp02 any updates? Let me know or I'll have to unassign you.

Yes sir I'm working on it due to my submission It is a little delayed
